server/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -113,6 +115,7 @@ private ContainerTag<?> renderBlockWithSubmitForm(Params params) {
         .withAction(formAction)
         .withMethod(HttpVerbs.POST)
         .with(makeCsrfTokenInputTag(params.request()))
+        .with(requiredFieldsExplanationContent(params.messages()))
```

Comment:
Ah I noticed the file screenshot is missing this text. Can you also edit the file upload logic (line 101)?

Reply:
Added a duplicate method in fileUploadViewStrategy but it feels redundant to have identical methods twice. What would be the best shared location for requiredFieldsExplanationContent()?

Reply:
Seems reasonable to me to put it in the parent class (ApplicationBaseView)